src/properties/Messages.ts

```diff
@@ -30,7 +30,11 @@ const MutableMessages = {
         },
         splunk: {
             page: {
-                title: 'Splunk Setup'
+                title: 'Red Hat Insights application for Splunk',
+                description: 'To finish the Splunk configuration, follow the instructions to start the automation process.',
+                help: 'Red Hat Insights application for Splunk configure the integration between your Splunk instance' +
+                      ' and Red Hat Insights to allow you to receive events from Insights.',
+                helpUrl: null
```

Comment:
was this left as null intentionally?

Reply:
Yes. Once we'll know what to include there, it can be added there.